angular-client/src/components/more-details/more-details.component.html

```diff
@@ -0,0 +1,3 @@
+<router-outlet></router-outlet>
```

Comment:
why do you need this?

Reply:
i do not remember why i put that it is def not needed